CHANGELOG.md

```diff
@@ -9,6 +9,25 @@ This changelog attempts to conform to the changelog spec on [keepachangelog.org]
 ## [Unreleased]
 Branch: `nilrt/master/sumo`
 
+
+## 8.7
+Branch: `nilrt/20.7/sumo`
+
+### meta-nilrt
+
+#### Added
+- [Added](https://github.com/ni/meta-nilrt/pull/63) an unsupported preview of newer kernel bases (currently 5.10) via the `kernel-next` recipe.
```

Comment:
Should we mention the package group here instead of the recipe? i.e. `packagegroup-ni-next-kernel`
It was introduced a while ago by this commit (so it might not fit under 8.7):
- https://github.com/ni/meta-nilrt/commit/4e075099adf828d17fd9c111c043efdba55247f1
and then extended by the 'kernel-devsrc' you've mentioned:
- https://github.com/ni/meta-nilrt/commit/0ce447888ee09add1a8f83656b102e32d7af50a5

Basically instead of mentioning `via the kernel-next recipe` say `via the packagegroup-ni-next-kernel ...`
It would provide a better starting point for somebody wanting to try the upcoming kernel.

Reply:
Makes sense; will fix.